WalletWasabi/WebClients/Wasabi/WasabiClient.cs

```diff
@@ -201,6 +201,11 @@ public async Task<UpdateStatus> CheckUpdatesAsync(CancellationToken cancel)
 			var clientUpToDate = Constants.ClientVersion >= versions.ClientVersion; // If the client version locally is greater than or equal to the backend's reported client version, then good.
 			var backendCompatible = int.Parse(Constants.BackendMajorVersion) == versions.BackendMajorVersion; // If the backend major and the client major are equal, then our softwares are compatible.
 
+			if (!backendCompatible)
```

Comment:
@nopara73 @lontivero 
Is this true? If the backend is not compatible the client must be outdated too, right?